packages/realm/package.json

```diff
@@ -196,6 +242,8 @@
     "module_path": "generated/ts",
     "host": "https://static.realm.io",
     "remote_path": "realm-js-prebuilds/{version}",
-    "napi_versions": [6]
+    "napi_versions": [
+      6
+    ]
```

Comment:
These format updates happened while testing the `npm version` command. I thought I might as well leave it in.